tmail-backend/integration-tests/jmap/memory-jmap-integration-tests/src/test/java/com/linagora/tmail/james/MemoryLinagoraEncryptedEmailFastViewGetMethodTest.java

```diff
@@ -0,0 +1,52 @@
+package com.linagora.tmail.james;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+import org.apache.james.JamesServerBuilder;
+import org.apache.james.JamesServerExtension;
+import org.apache.james.mailbox.inmemory.InMemoryMessageId;
+import org.apache.james.mailbox.model.MessageId;
+import org.apache.james.modules.TestJMAPServerModule;
+import org.apache.james.utils.GuiceProbe;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Scopes;
+import com.google.inject.multibindings.Multibinder;
+import com.linagora.tmail.encrypted.EncryptedEmailContentStore;
+import com.linagora.tmail.encrypted.InMemoryEncryptedEmailContentStore;
+import com.linagora.tmail.james.app.MemoryConfiguration;
+import com.linagora.tmail.james.app.MemoryServer;
+import com.linagora.tmail.james.common.LinagoraEncryptedEmailFastViewGetMethodContract;
+import com.linagora.tmail.james.common.probe.JmapGuiceEncryptedEmailContentStoreProbe;
+
+public class MemoryLinagoraEncryptedEmailFastViewGetMethodTest implements LinagoraEncryptedEmailFastViewGetMethodContract {
+
+    @RegisterExtension
+    static JamesServerExtension
+        jamesServerExtension = new JamesServerBuilder<MemoryConfiguration>(tmpDir ->
+        MemoryConfiguration.builder()
+            .workingDirectory(tmpDir)
+            .configurationFromClasspath()
+            .build())
+        .server(configuration -> MemoryServer.createServer(configuration)
+            .overrideWith(new TestJMAPServerModule())
+            .overrideWith(new InMemoryEncryptedEmailContentStoreModule())
+            .overrideWith(binder -> Multibinder.newSetBinder(binder, GuiceProbe.class)
+                .addBinding()
+                .to(JmapGuiceEncryptedEmailContentStoreProbe.class)))
+        .build();
+
+    @Override
+    public MessageId randomMessageId() {
+        return InMemoryMessageId.of(ThreadLocalRandom.current().nextInt(100000) + 100);
+    }
+}
+
+class InMemoryEncryptedEmailContentStoreModule extends AbstractModule {
```

Comment:
This module should not be bound in a test but should be packaged as part of the application.